.github/workflows/github-cxx-qt-tests.yml

```diff
@@ -422,6 +429,8 @@ jobs:
         SCCACHE_CACHE_SIZE: 600M
         SCCACHE_LOG: debug
         SCCACHE_ERROR_LOG: ${{ matrix.sccache_log_path }}
+        # Ensure that we share the same target sub folder
```

Comment:
Wonder how much sharing there can be between the builds, or if they are ultimately different anyway and need separate folders/caches. (cmake vs cargo)